sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/CheckpointStore/BlobsCheckpointStoreLiveTests.cs

```diff
@@ -77,29 +78,28 @@ await using (StorageScope storageScope = await StorageScope.CreateAsync())
         [Test]
         [TestCase(null)]
         [TestCase("ETag")]
```

Comment:
I considered that and ultimately left it because the Blobs store is using eTag values as the version.    Not that there's really a difference functionally, but I think it makes more sense in context.